src/Pages/Login.js

```diff
@@ -1,16 +1,74 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
 import logo from '../trivia.png';
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      name: '',
+      email: '',
+    };
+
+    this.validateFields = this.validateFields.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  validateFields() {
+    const { name, email } = this.state;
+    return (name.length > 0 && (/^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email)));
+  }
+
+  handleChange({ target }) {
+    const { value } = target;
+    this.setState({ [target.name]: value });
```

Comment:
```suggestion
    const { value, name } = target;
    this.setState({ [name]: value });
```
O código tá correto, seria mais por questão de padronização, já que o value já está sendo desestruturado.